internals/secrethub/run.go

```diff
@@ -438,55 +475,41 @@ func parseEnv(raw string) (map[string]tpl.VarTemplate, error) {
 		key := parts[0]
 		value := parts[1]
 
-		t, err := tpl.NewParser().Parse(value)
-		if err != nil {
-			return nil, ErrTemplate(i, err)
-		}
-
-		err = validation.ValidateEnvarName(key)
-		if err != nil {
-			return nil, ErrTemplate(i, err)
-		}
+		vars[key] = envvar{
+			key:        key,
+			value:      value,
+			lineNumber: i}
```

Comment:
Tiny, tiny thing, but if you're going to do multiline struct declarations, which I actually like, then please put the closing bracket on a new line as well for readability:

```
   lineNumber: i,
}
```